### PR TITLE
Azure Data Studio 1.31.0 Extension API

### PR DESCRIPTION
**Pull request by Charles-Gagnon**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: <<url here>>
- [x] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.

### CODE REVIEW

**Review by kburtram — APPROVED** [review]
LGTM.

**Comment by Charles-Gagnon** [comment]
@kburtram @alanrenmsft @kenvanhyning Could one of you please sign off on this?

**Review by alanrenmsft — APPROVED** [review]

File: types/azdata/tslint.json [hunk 1]
```diff
@@ -10,6 +10,7 @@
         "no-single-declare-module": false,
         "no-unnecessary-generics": false,
         "npm-naming": false,
-        "strict-export-declare-modifiers": false
+        "strict-export-declare-modifiers": false,
+        "unified-signatures": false
```
Comment (sandersn) [inline types/azdata/tslint.json:14]: Seems like unified-signatures should add an exception when any of the signatures is deprecated.

Comment (Charles-Gagnon) [reply types/azdata/tslint.json:14]: Agreed, thank you for submitting that!

**Pull request merged**